.ci/update-beats.groovy

```diff
@@ -0,0 +1,104 @@
+// Licensed to Elasticsearch B.V. under one or more contributor
+// license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright
+// ownership. Elasticsearch B.V. licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+// https://apm-ci.elastic.co/job/apm-server/job/update-json-schema-mbp/
+
+@Library('apm@current') _
+
+pipeline {
+  agent { label 'linux && immutable' }
+  environment {
+    REPO = 'apm-server'
+    BASE_DIR = "src/github.com/elastic/${env.REPO}"
+    HOME = "${env.WORKSPACE}"
+    NOTIFY_TO = credentials('notify-to')
+    JOB_GCS_BUCKET = credentials('gcs-bucket')
+    JOB_GIT_CREDENTIALS = "f6c7695a-671e-4f4f-a331-acdce44ff9ba"
+    PIPELINE_LOG_LEVEL = 'INFO'
+  }
+  triggers {
+    // Only master branch will run on a timer basis
+    cron(env.BRANCH_NAME == 'master' ? 'H H(4-5) * * 1,5' : '')
```

Comment:
doesn't this need to get updated to the `7.17` branch?

Reply:
That's right! 

```suggestion
    cron(env.BRANCH_NAME == '7.17' ? 'H H(4-5) * * 1,5' : '')
```

Reply:
https://github.com/elastic/apm-server/pull/7185 should be now the one in charge of deciding what branches should be updated